src/main/context.ts

```diff
@@ -2,7 +2,7 @@
 import { Event, EventHint, SdkInfo } from '@sentry/core';
 import { NodeClient } from '@sentry/node';
 import { app } from 'electron';
-import { SDK_VERSION } from './version';
+import { SDK_VERSION } from './version.js';
```

Comment:
does this get enforced via linting? If not, we should lint for this somehow.

also, do we also have to do this for the node and browser sdks? because that's what electron uses under the hood.

Reply:
I couldn't reproduce this issue with the Node SDK and I couldn't work out why it doesn't affect it. 

When we migrate the package to be ESM first (ie. `"type": "module"` in the package.json I think this will be a requirement for all packages.

I'll look into linting!